doc/example_query_logger.md

```diff
@@ -0,0 +1,90 @@
+2020-03-25 13:09:00 CET,,SUCCEEDED,UPDATE "standard1" SET "C0" = ?,"C1" = ?,"C2" = ?,"C3" = ?,"C4" = ? WHERE KEY=?[0x5ff08459ccd892a25f9189377eeee2a4f03827ef081b0c5909420a7ed9ad649929ad, 0x208ee47f58287238307277e66e81862e33bdd2e524afcf51eeb27516bb4a84cc5969, 0x3c4cb4956a93859adf04c0039d5c2ee0572d2504a8ba50d9f5dbae549910a9de5959, 0xdbf122dca6ed0e644b3691ea3ece67b48de7867e5b866efbada244e58ccba22c52ca, 0x609a5636d027d4e27cad7c3da7587e9cb197028a4408e3ac4de1e3ad11e0f29e9e9a, 0x4f503030314c35393330]
+2020-03-25 13:09:00 CET,,SUCCEEDED,UPDATE "standard1" SET "C0" = ?,"C1" = ?,"C2" = ?,"C3" = ?,"C4" = ? WHERE KEY=?[0xdc9e1bf05eab897f470a0777e3e3832852175016c58597038029ad7e2bc54d24086a, 0xcc4010fda2b4d80fbdb900134d27bf287d97df31875b848934d875a7ba248860a36f, 0xd82dda78d34b0d1e4d1b54c8140213b390476eee11e59899d74576bb468b544647a3, 0xbf1723eb11b1acdf9e97a86b0c2fb7fb75632f353c77b0ee521f37b37605d1d55d6d, 0x7cd7e4548d69eba1b5947858a8139da29447e41f02fc09741bd21a5089f04ec4b042, 0x37373539364d4f323330]
+2020-03-25 13:09:00 CET,,SUCCEEDED,UPDATE "standard1" SET "C0" = ?,"C1" = ?,"C2" = ?,"C3" = ?,"C4" = ? WHERE KEY=?[0x85182fdfe7f86306cd58ce81aa53cca24c193841d6ae8a91646a4c304dd9f47892f9, 0x1f5c9f8c30a62ce4815e936de814514ea33ed4d85fa33f90222d58a5a0a49b38d2fe, 0x225510932bd8812b4610817653b79c86552c99aefb177f4cefa4850a9139db696e6e, 0xd7e099a59da559e039600f6e2c0ede94bada8cfe5d8d4ff26d68ff597d3b3f68f2b4, 0xfbdd9cec024fcbde438399670475aa8900d80e0ed3d0c3312d9d4ae79241214d74c3, 0x30503337373039503231]
+2020-03-25 13:09:00 CET,,SUCCEEDED,UPDATE "standard1" SET "C0" = ?,"C1" = ?,"C2" = ?,"C3" = ?,"C4" = ? WHERE KEY=?[0x5c7310ecff5c4121973da9cacfdd5022dd27b9047bed9c1880fa881a91d4275ad020, 0xfa28e70b2dacef5f8bed360e7317018e789944fb8bc288224867d11d87463a3751ba, 0x78c72240e3bef9f89d0cd59e5b9c87cb2b323a0e19090b3495cbd0ba6703717560af, 0x6508112e2894adc8f4525fff28988061094b3f06570693ea36b611c4f78cfed0720e, 0xf5b3bd9baa993daac0b25ea274d6669eae349fc89ec0c6b761caa0b0a01e2bc45742, 0x374f324e363531323730]
+2020-03-25 13:09:00 CET,,SUCCEEDED,UPDATE "standard1" SET "C0" = ?,"C1" = ?,"C2" = ?,"C3" = ?,"C4" = ? WHERE KEY=?[0x3f0debec2d8107c992436dd8a3cb67ed21f4334b3b7b2c14b1dac6a59fa5870f75ab, 0x2835fb5492819b1a97cfbd478983ca4ca188a6e4c4aec74d7d6734e2a43f907a70cd, 0xc902e1eb11298a5b5e5c0d3bc5b5b2661091bc2b7f489625b7e5bcb847a1473aa5f2, 0xed2eae78d83e17a2a824e14b99b4644150529790cbbbad3ce65a2d91d93d8246175e, 0x554a0b119dedf8cd043c23e0a4c0d90ebc4aa01dc173d75758902c50706f46f86edd, 0x313637364d364b4c3630]
+2020-03-25 13:09:00 CET,,SUCCEEDED,UPDATE "standard1" SET "C0" = ?,"C1" = ?,"C2" = ?,"C3" = ?,"C4" = ? WHERE KEY=?[0xde7618ecf0e1cf11a43b9dbbb5fd7d8f4f4613a599ddf3495e8a2895de814dec3f28, 0x85580d850d89384bb20d251695c02dd926c0388731c0c96fabcf775d65c71d13427b, 0x5fea2011446e9ef66a25d89bea6123a4e9103c1f93ed059b230b482c4a4c087ca62f, 0x8447a04fdb26ff71088c8fcc25bc633c5fa86b9da879afdf141c517be32675d89c9b, 0xc61be8e74cb57a35c0626e1f48043ef53d06f5ed71288a213142c15372ee0e2c078a, 0x314e344b4d504d4b4b30]
```

Comment:
The example entries (from cassandra-stress) are so "complex"/boring to look at... With the blob values... And they all look the same...
It would be nice to have some "simpler entries" demonstrating the power of ecAudit... Maybe 1 select statement, 1 insert and 1 with prepared values...

Reply:
True. I'll fix that!